clusterloader2/testing/density/100_nodes/constraints.yaml

```diff
@@ -14,7 +14,7 @@ kube-controller-manager:
   cpuConstraint: 1.1
   memoryConstraint: 314572800 #300 * (1024 * 1024)
 kube-proxy:
-  cpuConstraint: 0.05
+  cpuConstraint: 0.06
```

Comment:
I was advocating for it in the past, but I think going to such low numbers doesn't make sense.
Can we make it 0.1 - I don't think going below that level of accuracy makes much sense.

WDYT?

Reply:
Agreed.

Added a comment in this file to make it clear for future edits. 
PTAL.